packages/kbn-optimizer/limits.yml

```diff
@@ -35,7 +35,7 @@ pageLoadAssetSize:
   indexLifecycleManagement: 107090
   indexManagement: 140608
   indexPatternManagement: 154222
-  infra: 197873
+  infra: 204800
```

Comment:
200k isn't terrible, but it's double our current goal for each plugin so it would be ideal if we could move some things to be async imported rather than raising the limit. Here is a diff I made to show what's new in the bundle to require this limit raise: https://gist.github.com/spalger/6bca361e5142e1a775a1ebb0797cebc4

I think that if you move a couple things around you could async import `log_stream_embeddable.tsx` and its datemath dependency. Would you mind giving that a shot to see if we can avoid raising the limit?

Reply:
@spalger thanks for reaching out! There are some other improvements that we can do in our main bundle and I have already created [a separate issue for it](https://github.com/elastic/kibana/issues/89025). If you agree, I'd like to increase the limit in the PR to move forward, with the promise to address it later.